src/Module.cpp

```diff
@@ -49,7 +49,10 @@ std::map<OutputFileType, const OutputInfo> get_output_info(const Target &target)
         {OutputFileType::schedule, {"schedule", ".schedule.h", IsSingle}},
         {OutputFileType::static_library, {"static_library", is_windows_coff ? ".lib" : ".a", IsSingle}},
         {OutputFileType::stmt, {"stmt", ".stmt", IsMulti}},
+        {OutputFileType::conceptual_stmt, {"coneptual_stmt", ".conceptual.stmt", IsMulti}},
```

Comment:
Is it a typo? `coneptual -> conceptual`?

```
Unhandled exception: Error: Unrecognized emit option: conceptual_stmt is not one of [assembly bitcode c_header c_source compiler_log cpp_stub featurization function_info_header hlpipe llvm_assembly object python_extension pytorch_wrapper registration schedule static_library stmt coneptual_stmt stmt_html conceptual_stmt_html device_code], ignoring.
```

Reply:
Thanks for spotting this. My light form of dyslexia makes spotting these hard.